Project-I-Social-Card/social-card/src/components/CardComponents/CardBanner.js

```diff
@@ -1,2 +1,10 @@
 import React from 'react';
 import './Card.css';
+
+const CardBanner = () => {
+    return (
+        <img class='background' src="https://tk-assets.lambdaschool.com/fcd75197-7d12-46ec-bc9e-4130f34822fa_reactbackground.png" />
```

Comment:
An *alt* attribute can be added to the img element.
https://www.w3schools.com/tags/att_img_alt.asp